libs/community/langchain_community/cache.py

```diff
@@ -349,21 +350,69 @@ def clear(self, **kwargs: Any) -> None:
         self.redis.flushdb(flush_type=asynchronous)
 
 
-class RedisCache(BaseCache):
-    """Cache that uses Redis as a backend."""
+class RedisCacheBase(BaseCache, ABC):
```

Comment:
Could we mark this as private?

Reply:
Sure, can you show me the example? Something like `__RedisCacheBase` ?

Reply:
Looked at other examples in codebase and seems like the pattern is to use a single underscore, so I pushed `_RedisCacheBase` rename.